vividus-tests/src/main/resources/properties/suite/failFast/suite.properties

```diff
@@ -1,8 +1,8 @@
 bdd.story-loader.batch-1.resource-location=story/integration/failFast
-bdd.story-loader.batch-1.resource-include-patterns=FailFast*.story,Success.story
+bdd.story-loader.batch-1.resource-include-patterns=FailFast*.story ,Success.story
 bdd.batch-1.threads=2
 bdd.story-loader.batch-2.resource-location=story/integration/failFast
-bdd.story-loader.batch-2.resource-include-patterns=Should not be executed.story
+bdd.story-loader.batch-2.resource-include-patterns= Should not be executed.story
```

Comment:
it seems this batch shouldn't be executed, so it's not a good place to put spaces